<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdlenessWatcherSettings</class>
 <widget class="QWidget" name="IdlenessWatcherSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>452</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="idlenessWatcherGroupBox">
     <property name="title">
      <string>Enab&amp;le Idleness Watcher</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="idleTimeACLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>When idle on AC:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="idleACActionComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="idleACActionLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>AC Idle time:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTimeEdit" name="idleACTimeEdit">
          <property name="toolTip">
           <string>Minutes:Seconds (min: 01:30)</string>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>1</minute>
            <second>0</second>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="idleTimeBatteryLabel">
          <property name="text">
           <string>When idle on Battery:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="idleBatteryActionComboBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="idleBatteryActionLabel">
          <property name="text">
           <string>Battery Idle time:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QTimeEdit" name="idleBatteryTimeEdit">
          <property name="toolTip">
           <string>Minutes:Seconds (min: 01:30)</string>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>1</minute>
            <second>0</second>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>mm:ss</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="idlenessBacklightWatcherGroupBox">
        <property name="title">
         <string>Enable Backlight Change</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="backlightLabel">
           <property name="text">
            <string>Backlight (%):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="backlightSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="idleTimeBacklightLabel">
           <property name="text">
            <string>Idle time:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QTimeEdit" name="idleTimeBacklightTimeEdit">
           <property name="toolTip">
            <string>Minutes:Seconds</string>
           </property>
           <property name="maximumTime">
            <time>
             <hour>22</hour>
             <minute>59</minute>
             <second>59</second>
            </time>
           </property>
           <property name="minimumTime">
            <time>
             <hour>0</hour>
             <minute>0</minute>
             <second>1</second>
            </time>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::MinuteSection</enum>
           </property>
           <property name="displayFormat">
            <string>mm:ss</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="onBatteryDischargingCheckBox">
           <property name="text">
            <string>On battery discharging </string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="checkBacklightButton">
           <property name="text">
            <string>Check backlight</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="monitorGroupBox">
     <property name="title">
      <string>Command to Turn off Monitor</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Recommended command for:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="monitorComboBox"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="monitorLineEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="disableIdlenessFullscreenBox">
     <property name="text">
      <string>Disable idleness checks when fullscreen</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
